### DISCUSSION TITLE
ORCID iDs for non-members?

### DISCUSSION THREAD

**Discussion by CarolineWinter** [knowledgefutures/pubpub#2342] [Support]
Hello, I'd like to include clickable ORCID iDs in a publication byline, beside contributors' names. I know there's a field for ORCID iDs for contributors who are members, but it doesn't generate a clickable link, and those who aren't members can't list their iDs. Is there a way to add this info to their byline?

**Comment by gabestein** [comment]
Hi Caroline — clickable ORCID links are available in the "details" section of any pub, including for non-members added via the Pub contributors (see screenshots). We've resisted cluttering the byline itself with extra links and icons, but we are considering adding a modal that would display when you hover over an author that would include additional information like ORCIDs. Do you think that would satisfy your needs?

<img width="1039" alt="Screenshot 2022-11-14 at 14 03 21" src="https://user-images.githubusercontent.com/639110/201744090-54fac351-b495-4c06-a282-bfbefd137bb9.png">

<img width="982" alt="Screenshot 2022-11-14 at 14 04 19" src="https://user-images.githubusercontent.com/639110/201744235-37e4b750-ceba-4a4f-bdcb-ef44f1f40edb.png">

**Reply by CarolineWinter** [reply]
Thanks, Gabriel! I'll give that a try. And yes, displaying the ORCID iD somehow would be good, but a clickable link near the contributor's name would be ideal.
